[2025-05-20T21:41:57.423+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api permission to 509
[2025-05-20T21:41:57.424+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api'
[2025-05-20T21:41:57.451+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api permission to 509
[2025-05-20T21:41:57.452+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api'
[2025-05-20T21:41:57.485+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.extract_api manual__2025-05-20T21:41:56.016288+00:00 [queued]>
[2025-05-20T21:41:57.491+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.extract_api manual__2025-05-20T21:41:56.016288+00:00 [queued]>
[2025-05-20T21:41:57.491+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-20T21:41:57.502+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): extract_api> on 2025-05-20 21:41:56.016288+00:00
[2025-05-20T21:41:57.506+0000] {standard_task_runner.py:60} INFO - Started process 223 to run task
[2025-05-20T21:41:57.508+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'extract_api', 'manual__2025-05-20T21:41:56.016288+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmp46q2cthj']
[2025-05-20T21:41:57.510+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask extract_api
[2025-05-20T21:41:57.552+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api permission to 509
[2025-05-20T21:41:57.552+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-20T21:41:56.016288+00:00/task_id=extract_api'
[2025-05-20T21:41:57.553+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.extract_api manual__2025-05-20T21:41:56.016288+00:00 [running]> on host b22c2b86ecca
[2025-05-20T21:41:57.765+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='extract_api' AIRFLOW_CTX_EXECUTION_DATE='2025-05-20T21:41:56.016288+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-20T21:41:56.016288+00:00'
[2025-05-20T21:41:58.494+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-20T21:41:58.503+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=extract_api, execution_date=20250520T214156, start_date=20250520T214157, end_date=20250520T214158
[2025-05-20T21:41:58.522+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-20T21:41:58.546+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
